internal/core/commitment_behavior.go

```diff
@@ -42,8 +42,14 @@ func (b CommitmentBehavior) Validate(path string) (errs errext.ErrorSet) {
 			errs.Addf("invalid value: %s.until_percent may not be bigger than 100", path)
 		}
 	}
+	if conversionRule, ok := b.ConversionRule.Unpack(); ok {
+		identifier = conversionRule.Identifier
+		if slices.Contains(occupiedConversionIdentifiers, conversionRule.Identifier) {
+			errs.Addf("invalid value: %s.conversion_rule.identifier must be unique, but %q is already used by another serviceType", path, conversionRule.Identifier)
```

Comment:
This phrasing is imprecise:
```suggestion
			errs.Addf("invalid value: %s.conversion_rule.identifier values must be restricted to a single serviceType, but %q is already used by another serviceType", path, conversionRule.Identifier)
```